src/test/java/com/crud/tasks/mapper/TrelloMapperTestSuite.java

```diff
@@ -53,11 +54,10 @@
     }
 
     @Test
-    public void mapToList() {
-        //Given
+    public void testMapToListMethod() {
```

Comment:
nie dawaj w nazwie metody „method” - po co to? Proponuje nazwe `shouldMapTrelloListToTrelloListDto` czy jakos tak